### PR TITLE
ON-12793 # Added hint to section element

### PR DESCRIPTION
**Pull request by Zaxist**

### CODE REVIEW

File: src/form-elements/FormElementSection.tsx [hunk 1]
```diff
@@ -19,20 +19,27 @@ function FormElementSection(
...
-          </i>
-        </div>
         <h3 className="ob-section__header-text title is-3">
           {props.element.label}
+          {props.element.hint && (
+            <Tooltip
+              title={props.element.hint}
+              arrow
+              enterTouchDelay={0}
+              leaveTouchDelay={10000}
```
Comment (mymattcarroll) [inline src/form-elements/FormElementSection.tsx:29]: was this value just for testing?

File: src/form-elements/FormElementSection.tsx [hunk 2]
```diff
@@ -19,20 +19,27 @@ function FormElementSection(
...
+              <i className="material-icons has-text-grey-light ob-label__hint">
+                info
+              </i>
+            </Tooltip>
+          )}
         </h3>
         <div className="ob-section__header-icon-container">
+          {!isValid && (
+            <i className="material-icons cypress-section-invalid-valid has-text-danger">
+              {'warning'}
```
Comment (mymattcarroll) [inline src/form-elements/FormElementSection.tsx:40]:
```suggestion
              warning
```

File: src/form-elements/FormElementSection.tsx [hunk 3]
```diff
@@ -19,20 +19,27 @@ function FormElementSection(
...
+            >
+              <i className="material-icons has-text-grey-light ob-label__hint">
+                info
+              </i>
+            </Tooltip>
+          )}
         </h3>
         <div className="ob-section__header-icon-container">
+          {!isValid && (
+            <i className="material-icons cypress-section-invalid-valid has-text-danger">
```
Comment (mymattcarroll) [inline src/form-elements/FormElementSection.tsx:39]:
```suggestion
            <i className="material-icons cypress-section-invalid has-text-danger">
```

File: src/styles/section.scss [hunk 4]
```diff
@@ -23,6 +23,10 @@
   display: flex;
   align-items: center;
   justify-content: center;
+
+  & > *:not(:last-child) {
+    margin-right: $size-8;
+  }
```
Comment (mymattcarroll) [inline src/styles/section.scss:29]: This is quite specific, meaning difficult to override.

Might be better to add another class and move these styles into that class and not make it nested.

File: src/form-elements/FormElementSection.tsx (hunk 1 above)
Comment (Zaxist) [reply src/form-elements/FormElementSection.tsx:29]: No, thats what the other hints have set

**Pull request merged**